shared/actions/wallets.js

```diff
@@ -10,6 +10,43 @@ import logger from '../logger'
 import type {TypedState} from '../constants/reducer'
 import {walletsTab} from '../constants/tabs'
 
+const buildPayment = (state: TypedState) =>
+  RPCTypes.localBuildPaymentLocalRpcPromise({
+    amount: state.wallets.get('buildingPayment').get('amount'),
+    // FIXME: Assumes XLM.
+    // currency: state.wallets.get('buildingPayment').get('currency'),
```

Comment:
`currency` is not outside assets, but something like USD.  See the avdl:

    string amount, // Opt*. Amount of the asset OR outside currency being sent. Example: "3.005"                      
    // Zero or one of `asset` and `currency` should be set.                                                           
    // If both are null or `asset` is native XLM then XLM are being sent.                                             
    // If `currency` is set then XLM is being sent after converting `amount`.                                         
    // Note: Sending non-XLM assets is not yet supported.                                                             
    union { null, OutsideCurrencyCode } currency,                                                                     
    union { null, Asset } asset,